irrd/scripts/irr_rpsl_submit.py

```diff
@@ -90,8 +599,11 @@ def extract_request_body(requests_text: str):
                 password = line.split(":", maxsplit=1)[1].strip()
                 passwords.append(password)
             elif line.startswith("override:"):
-                override = line.split(":", maxsplit=1)[1].strip()
-                overrides.append(override)
+                password = line.split(":", maxsplit=1)[1].strip()
+                logger.debug("override password is %s", password)
```

Comment:
This may only be a debug log, but I do not like any logging of cleartext passwords. Feels like the kind of thing someone will regret in 5 years.

```suggestion
                logger.debug("detected override password")
```

Reply:
The docstrings are there because pylint wanted them. They aren't hurting anything so I'd leave them alone.